src/index.css

```diff
@@ -1,27 +1,264 @@
-body {
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400&display=swap');
+
+* {
+  padding: 0;
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  box-sizing: border-box;
+}
+
+html {
```

Comment:
Requests based on above screenshot :)

- [ ] 1. Please fix-up margin error, on your left and right side. Your app shouldn't have scroll-x.
- [ ] 2. Kindly align add form with books list. You can see in screenshot add form is a bit longer. 
- [ ] 3. Put your navlinkst `Books` and `Categories` closer to Logo.
- [ ] 4. `UPDATE PROGRESS` in books list have more padding. Kindly add that as well.
- [ ] 5. 
 ```javascript
  <span>Chapter 1</span>
  <button type="button">UPDATE PROGRESS</button>
 ```  
They have some gap between them. Please add it as well :)

Reply:
- [ ] The last one this [link](https://github.com/Oscah01/book-store/issues) in your readme is broken. Kindly fix it as well.